src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -130,11 +137,30 @@ trait SharedFileSystem {
       WdlFile(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
+    def localizeInputToCallDirectory(path: String): Path = {
+      val originalPath = Paths.get(path)
+      val callPath = Try {
+        val callInputsDirectory = LocalBackend.hostCallPath(workflowDescriptor, call.name, callKey.index).resolve("inputs")
+        if (!callInputsDirectory.toFile.exists) callInputsDirectory.toFile.mkdirs
+        val hash = hashPath(originalPath)
+        callInputsDirectory.resolve(s"$hash-${originalPath.getFileName.toString}")
```

Comment:
Per DSDEEPB-1566 (a real problem Jose ran into), if a data file and its index come from different directories Cromwell currently prepends different hashes to the file names.  Cromwell does this to avoid colliding input files with the same file name in the workflow inputs directory (another real problem someone ran into).  Unfortunately the logic in some of our tools expects index files to have the same base name as the data file, and hash-prepending file names can break that.